packages/@atjson/source-gdocs-paste/src/converter.ts

```diff
@@ -149,6 +157,99 @@ GDocsSource.defineConverterTo(OffsetSource, (doc) => {
     );
   }
 
+  // Convert vertical tabs to line breaks
+  for (let table of doc.match(TABLE)) {
+    let id = uuid();
+    let rows: Array<Array<{ slice: string; jsonValue: string }>> = [];
+
+    // Gather the dataset info first
+    for (let tableRow of doc.match(TABLE_ROW, table.start, table.end)) {
+      let row: Array<{ slice: string; jsonValue: string }> = [];
+      doc.addAnnotations(
+        new ParseAnnotation({
+          start: tableRow.start,
+          end: tableRow.start + 1,
+        })
+      );
+
+      for (let tableCell of doc.match(
+        TABLE_CELL,
+        tableRow.start + 1,
+        tableRow.end
+      )) {
+        let slice = new SliceAnnotation({
+          start: tableCell.start,
+          end: tableCell.end,
+          attributes: {
+            refs: [id],
+          },
+        });
+        row.push({
+          slice: slice.id,
+          jsonValue: tableCell.matches[1],
+        });
+        doc.addAnnotations(
+          new ParseAnnotation({
+            start: tableCell.start,
+            end: tableCell.start + 1,
+          }),
+          slice,
+          new ParseAnnotation({
+            start: tableCell.end - 1,
+            end: tableCell.end,
+          })
+        );
+      }
+      rows.push(row);
+    }
+    let [header, ...body] = rows;
```

Comment:
cool, I was thinking about whether we should assume the first row is the column headers or assume there are no column headers, but this seems good to me

Reply:
yeah, I don't know how we would otherwise be able to tell tbh